includes/model/class-blog-user.php

```diff
@@ -0,0 +1,95 @@
+<?php
+namespace Activitypub\Model;
+
+use WP_Query;
+use Activitypub\User_Factory;
+
+class Blog_User extends User {
+	/**
+	 * The User-ID
+	 *
+	 * @var int
+	 */
+	public $user_id = User_Factory::BLOG_USER_ID;
+
+	/**
+	 * The User-Type
+	 *
+	 * @var string
+	 */
+	private $type = 'Person';
+
+	/**
+	 * The User constructor.
+	 *
+	 * @param int $user_id The User-ID.
+	 */
+	public function __construct( $user_id ) {
+		add_filter( 'activitypub_json_author_array', array( $this, 'add_api_endpoints' ), 10, 2 );
+	}
+
+	/**
+	 * Get the User-Name.
+	 *
+	 * @return string The User-Name.
+	 */
+	public function get_name() {
+		return \esc_html( \get_bloginfo( 'name' ) );
+	}
+
+	/**
+	 * Get the User-Description.
+	 *
+	 * @return string The User-Description.
+	 */
+	public function get_summary() {
+		return \wpautop( \wp_kses( \get_bloginfo( 'description' ), 'default' ) );
+	}
+
+	/**
+	 * Get the User-Url.
+	 *
+	 * @return string The User-Url.
+	 */
+	public function get_url() {
+		return \esc_url( \trailingslashit( get_home_url() ) . '@' . $this->get_username() );
+	}
+
+	public function get_username() {
+		return \esc_html( \get_bloginfo( 'activitypub_blog_identifier', 'feed' ) );
```

Comment:
Curious about the use of `get_bloginfo` here? (And the other one for `activitypub_application_identifier`)

Do we want `get_option()`? `get_bloginfo` is just a proxy for a variety of standard site options.

Reply:
Oops! That was a copy and paste issue! I wanted to use get_option! Good catch!